configure.ac

```diff
@@ -532,6 +533,45 @@ AS_IF([test "x${enable_opus_cdda}" != "xno"],
 
 AM_CONDITIONAL(USE_OPUS, test "${HAVE_OPUS}" = "yes")
 
+
+dnl FluidSynth software MIDI synthesiser
+dnl ------------------------------------
+dnl
+AH_TEMPLATE(C_FLUIDSYNTH, [Define to 1 to enable FluidSynth MIDI synthesiser])
+AC_ARG_ENABLE(fluidsynth,
+              AS_HELP_STRING([--disable-fluidsynth],
+                             [Disable FluidSynth integration]))
+
+if test "${enable_fluidsynth}" != "no" ; then
```

Comment:
Suggest (the admittedly nasty looking) x"string", x"$variable" syntax:

 `test x"${enable_fluidsynth}" != x"no"` 

And below.